From: Mans Rullgard <mans@mansr.com>
Date: Wed, 3 Sep 2008 23:17:06 +0000 (+0100)
Subject: OMAP: Fix unloading lcd_omap3beagle module
X-Git-Url: http://git.mansr.com/?p=linux-omap;a=commitdiff_plain;h=6db3a4c28c6819047362fc8ac460cf8c124c6074

OMAP: Fix unloading lcd_omap3beagle module

Unregister the panel when unloading the lcd_omap3beagle module.

Signed-off-by: Mans Rullgard <mans@mansr.com>
---

diff --git a/drivers/video/omap/lcd_omap3beagle.c b/drivers/video/omap/lcd_omap3beagle.c
index 7b63db9..32bd988 100644
--- a/drivers/video/omap/lcd_omap3beagle.c
+++ b/drivers/video/omap/lcd_omap3beagle.c
@@ -90,6 +90,7 @@ static int omap3beagle_panel_probe(struct platform_device *pdev)
 
 static int omap3beagle_panel_remove(struct platform_device *pdev)
 {
+	omapfb_unregister_panel(&omap3beagle_panel);
 	return 0;
 }
 
